modules/mobile/spec/request/veterans_affairs_eligibility_request_spec.rb

```diff
@@ -89,6 +89,12 @@
                'directEligibleFacilities' => [] }]
           )
         end
+
+        it 'does not include facility in cc_supported ids' do
+          cc_supported_facility_ids = response.parsed_body.dig('data', 'attributes', 'ccSupported')
+
+          expect(cc_supported_facility_ids).not_to eq(%w[489])
```

Comment:
It looks like 489 is the only facility id it could be, but in general I would argue it might be better to write this in the positive `.to eq([])` instead of testing absence. Just in case there's some other unexpected value there.

Reply:
That's understandable. In my head, I was thinking tests might change and checking for not equal would accommodate if the test were run in a multiple facility situation. That was pointless to account for though because the context won't change. Updated in the latest commit.